linux_os/guide/system/network/network-firewalld/ruleset_modifications/ensure_firewall_rules_for_open_ports/oval/shared.xml

```diff
@@ -60,17 +74,104 @@
     <linux:local_port datatype="int" operation="greater than or equal">0</linux:local_port>
   </linux:inetlisteningservers_state>
 
+  <local_variable id="var_firewalld_active_zones" version="1" comment="Firewalled ports according to firewalld configuration" datatype="string">
```

Comment:
Could you ensure the lines length are aligned to the Project Style Guide, please?